composer.json

```diff
@@ -11,7 +11,10 @@
         "zendframework"
     ],
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "platform": {
+            "php": "7.3"
```

Comment:
Maybe we should use the specific version we currently have in docker. It is because some packages might require specific version of PHP 7.3 (for example symfony 5.0 requires PHP 7.2.5, not just 7.3 - see: https://github.com/symfony/symfony/blob/c699b9cb3d16bce06871542c12d833044c4ce7dc/composer.json#L19)

Reply:
Actually, not just docker, but the one in `.platform.app.yml`, as _that is the deployment platform_. They all need to be coordinated.

Reply:
symfony requires 7.2.5 because they need a fix from that version. I dont think this is used because they had a deployment server running that version.